app/package.json

```
@@ -24,7 +24,7 @@
     "server": "cd server && npm start",
     "start": "concurrently --kc \"blue.dim,green.dim\" \"npm:server\" \"npm run client\"",
     "setup": "echo \"Installing project dependencies...\" && npm ci && echo \"Installing client dependencies...\" && cd client && npm ci --legacy-peer-deps && echo \"Installing server dependencies...\" && cd .. && cd server && npm ci",
-    "cypress": "concurrently -kc \"blue.dim,green.dim,yellow.dim\" -n server,client,cypress \"npm:server\" \"npm:client\" \"cypress open --env coverage=false\"",
+    "cypress": "concurrently -kc \"blue.dim,green.dim,yellow.dim\" -n server,client,cypress \"npm:server\" \"npm:client\" \"cypress open --env coverage=true\"",
```

```suggestion
    "cypress": "concurrently -kc \"blue.dim,green.dim,yellow.dim\" -n server,client,cypress \"npm:server\" \"npm:client\" \"cypress open --env coverage=false\"",
```

I'm going to go ahead and undo this.

We have coverage set to false to avoid out of memory errors. It might not be a problem now, but as we add more tests we could start getting out of memory errors.